src/app/components/table/table.component.html

```diff
@@ -4,35 +4,35 @@
         <div class="label">Selected: </div>
         <div class="data_label">{{label || ''}}</div>
         <div class="edge_label" *ngIf="edgeLabel">{{edgeLabel}}</div>
-        <div class="tier_label" (click)="$event.stopPropagation(); deselectTier()">
-            <ng-container *ngIf="phylogenySelectedTier">
-                {{phylogenySelectedTier}} <div class="close_icon"></div>
+        <div class="tier_label" (click)="$event.stopPropagation(); deselectConsequence()">
+            <ng-container *ngIf="consequence">
+                {{ consequence }} <div class="close_icon"></div>
             </ng-container>
-            <ng-container *ngIf="phylogenySelectedTier">
-                All Tiers
+            <ng-container *ngIf="!consequence">
+                All Consequences
             </ng-container>
         </div>
     </div>
 
     <table>
         <thead>
             <tr>
-                <th *ngFor="let column of tableColumns" [ngClass]="'column_' + column.field">
+                <th *ngFor="let column of tableColumns" [ngClass]="'column_' + column.cssSuffix">
                     {{column.label}}
                 </th>
             </tr>
         </thead>
 
         <tbody>
-            <ng-container *ngFor="let data of detailsData">
-                <tr *ngIf="!phylogenySelectedTier || phylogenySelectedTier === data.tier" (click)="$event.stopPropagation(); selectRow(data)">
-                    <td *ngFor="let column of tableColumns; index as i" [ngClass]="'column_' + column.field">
-                        {{data[column.field]}}
+            <ng-container *ngFor="let row of tableRows">
+                <tr *ngIf="!consequence || consequence === row.variant.consequence" (click)="$event.stopPropagation(); selectRow(row)">
```

Comment:
this table gets big enough to be slow; we can chat about adding a table from one of our favorite component libraries for pagination, etc.